config/settings.yaml.example

```diff
@@ -71,6 +71,24 @@
 #  :dynflow:
 #    :enabled: true
 
+# before enabling any telemetry install foreman-telemetry package (telemetry bundler group)
+:telemetry:
```

Comment:
What is the default state? Publishing stats to the logger?

Reply:
Default state is disabled completly. I renamed `rails` to `logging` now, hope it's more clear. You should see three `enabled: false` lines in this block.